src/app/data-planes/data/index.ts

```diff
@@ -52,3 +53,28 @@ export function getStatusAndReason(dataplaneOverview: DataplaneOverview): { stat
     reason: unhealthyInbounds,
   }
 }
+
+export function getTags(dataplaneOverview: DataplaneOverview): LabelValue[] {
+  let tags: string[] = []
+  const separator = '='
+  const { gateway, inbound } = dataplaneOverview.dataplane.networking
+
+  if (inbound) {
+    tags = inbound
+      .flatMap((inbound) => Object.entries(inbound.tags))
+      .map(([key, value]) => key + separator + value)
+  }
+
+  if (gateway) {
+    tags = Object.entries(gateway.tags).map(([key, value]) => key + separator + value)
```

Comment:
Do you have a reason why you would prefer/change these to use addition here?

If not, could we switch these back to use backticks/template literals instead of addition, especially seeing as this is what it was using before.

Reply:
This looks like addition to me, these are strings, not numbers. We should use template literals always for joining strings, for one it prevents `valueOf` being used instead of `toString`.

We shouldn't get into the habit of thinking addition for string joining is ok and there is no reason to change this here.

Reply:
I’ve changed it because it really doesn’t matter, but please consider whether this kind of suggestion is really worth the time spent. The plus operator is extremely well-established to do string concatenation in JavaScript and especially since we’re dealing with a pretty well-typed code base, we actually know that none of the values here can even be numbers in theory.

Confusing an expression involving strings and the plus operator with number addition is not a problem in our code base because we’ve established better knowledge about the values at play.